CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_shopify_source v0.18.1
+
+[PR #103](https://github.com/fivetran/dbt_shopify_source/pull/103) includes the following updates:
+
+## Under the Hood
+- Added BigQuery JSON field support for the following model and columns:
+  - `stg_shopify__order`: `total_shipping_price_set` column
+  - `stg_shopify__transaction`: `receipt` column
+  - `stg_shopify__collection`: `rules` column
+- Introduced the new `json_to_string()` macro to be used in the BigQuery JSON field support.
```

Comment:
The name of the macro sorta already implies this, but I wonder if we should more explicitly call out that we are converting the JSON fields to strings, in case people are expecting to see JSONs

Reply:
Agreed, updated in the latest commit